site/content/3.13/graphs/Graph_visualizer/_index.md

```diff
@@ -0,0 +1,164 @@
+
+### List All Graphs
+  Arango Db provides various types of graphs like
+- General Graphs
+- SmartGraphs
+- SatelliteGraphs
+- EnterpriseGraphs
+- KnowledgeGraphs
+
+![Graphs List](../../../images/Graph_VisualizerList_All_Graphs.png)
+
+### Select and Load a Graph
+
+After selecting a graph from the list, it may not be immediately visualized on the canvas. In cases such as:
+- Using the **Clear Canvas** option
+- Reopening the Graph Visualizer after a previous session
+- Selecting a graph with no initial nodes displayed
+
+You may see a message prompting you to use the **Explore** button.  
+To view the graph:
+
+- Click **Explore** and search for a node by name or ID.
+- Alternatively, use the **New Query** or **Saved Query** buttons to display part of the graph.
+
+![Loaded Graph](../../../images/Graph_visuaizer_Load_Graph.png)
+
+{{< tip >}}
+Use the **Explore** button to run AQL queries and render results directly on the graph.
+You can also save your queries for future use.
+{{< /tip >}}
+## Search & Filter Data
+
+The top-left section of the Graph Visualizer includes powerful search and query tools for interactive exploration.
+
+### Search
+
+1. Click the **Search** icon.
+2. Choose a vertex type.
+3. Enter a field or property to search for specific nodes.
+
+Enter the DataValue or fields to get the required Data
+![Search](../../../images/Graph_Visualizer_Search_Response.png)
+
+### Saved Queries
+
+- Open **Search** and click **Saved Queries**.
+- Each query has options to:
+  - **Run** it again
+  - **Copy** to modify
+  - **Delete** if no longer needed
+
+![Saved Queries](../../../images/Graph_Visualizer_savedquery.png)
+
+### New Queries
+- Go to **Search** → **New Query**.
+- Write and run your own AQL query.
+- Results are shown on the graph.
+- You can save it with a custom name and description.
+
+![](../../../images/Graph_Visualizer_new_Query.png)
+
+- **Response of the NewQuery**
+![](../../../images/Graph_Visualizer_NewQuery_response.png)
+
+
+## Visual Customization
+
+Click the **style panel** (bottom-right) to:
+- Change node/edge colors
+- Adjust opacity
+
+### Styling Options 
+Modify the **color** and **opacity** of selected nodes or edges for emphasis or categorization.
+![](../../../images/Graph_Visualizer_Styling_Options.png)
+
+### Edge-Specific Options
+
+- Set line thickness to represent weights.
+- Choose different arrowhead styles for source and target nodes.
+
+![](../../../images/Graph_Visualizer_Edge-Specific_Options.PNG)
+
+## Layouts and Navigation Tools
+
+**Graph Layout Tools**:
+
+- **Mini-map**: Small overview to navigate the graph.
+![](../../../images/Graph_Visualizer_minimap.PNG)
+
+- **Zoom Controls**: Zoom in/out or set specific zoom.
+
+- **Fit to Screen**: Resize and center the graph view.
+
+- **Re-run Layout**: Automatically rearranges nodes.
+
+- **Layout Algorithms**: Choose between layouts to better see clusters or flows.
```

Comment:
How to apply the layout algorithm should be described in better detail, and the effects of each be described in a way that let's users make a good decision IMO